src/task/image.ts

```diff
@@ -0,0 +1,9 @@
+import { type User } from '../api/auth/authorization.js';
+import { ImageModel } from '../api/db/models/Image.js';
+import { TaskInput } from '../api/db/models/Task.js';
+import type * as gql from '../@types/graphql.js';
+
+export async function DeleteImages(task: TaskInput<gql.DeleteImagesInput>) {
+  const context = { user: { is_superuser: true, curr_project: task.projectId } as User };
+  return await ImageModel.deleteImages(task.config, context);
```

Comment:
Deciding if we should use the existing delete images function. Need to check why we are deleting images one at a time

Reply:
One at a time as opposed to like a bulkWrite() or [deleteMany()](https://www.mongodb.com/docs/manual/reference/method/db.collection.deleteMany/) kind of operation?

I think it's because each "image" consists of:
- the three versions of the image file stored in S3
- the `image` record itself, as well as a corresponding `imageAttempt` and `imageError` record

So we go one at a time to remove all of those pieces for each image so if something goes wrong we don't have partial data for some of the images.

Reply:
Yea makes sense after some thought. I think this will mean the async task will just end up calling original deleteImages